### PR TITLE
Remove StrictHostKey checking

### PR DESCRIPTION
**Pull request by cultavix**
This will allow to use the bastion host more transparently and the ability to use key-less logins (by using ssh-agent)

```
Host 10.*
    User ubuntu
    ProxyCommand ssh -o 'ForwardAgent yes' ubuntu@bastion.example.com 'ssh-add && nc %h %p'
```

### CODE REVIEW

File: user_data.sh [hunk 1]
```diff
@@ -18,6 +18,10 @@ pip install --upgrade awscli
 ##############
 
 cat <<"EOF" > /home/${ssh_user}/update_ssh_authorized_keys.sh
+cat <<EOF > /home/${ssh_user}/.ssh/config
+Host *
+    StrictHostKeyChecking no
+EOF
```
Comment (antonbabenko) [inline user_data.sh]: This block should be not inside line 20, but outside of file `update_ssh_authorized_keys.sh`.

Comment (cultavix) [inline user_data.sh]: ah ok yes, I get it :)

File: user_data.sh [hunk 2]
```diff
@@ -53,6 +49,12 @@ done
 chown $SSH_USER:$SSH_USER $KEYS_FILE
 chmod 600 $KEYS_FILE
 mv $TEMP_KEYS_FILE $KEYS_FILE
+
+cat <<"SSHCONFIG" > /home/${ssh_user}/.ssh/config
```
Comment (antonbabenko) [inline user_data.sh]: Same. Move these lines outside of file `update_ssh_authorized_keys.sh` (after "EOF" on line 58, for example) and set correct permissions to `.ssh/config` file (whatever it should be).

**Comment by cultavix** [comment]
@antonbabenko is that better? :)

**Comment by antonbabenko** [comment]
Thank you

**Pull request merged**